api/app.py

```diff
@@ -12,18 +12,15 @@
 import flask_cors
 
 
-_EXTENSION_ID = 'biiglkpcdjpendjobkhgoeflaejipmfg'
-
-
 app = flask.Flask(__name__)
 cors = flask_cors.CORS(app, resources={
```

Comment:
FYI: I guess we can just add localhost to host_permissions in manifest.json and don't need CORS headers?

Reply:
Thank you for checking this out.

> FYI: I guess we can just add localhost to host_permissions in manifest.json and don't need CORS headers?

Perhaps I am doing something unnecessary due to a misunderstanding or lack of understanding of `host_permissions` in `manifest.json` and CORS. Once I have a better understanding of these settings, I will remove the unnecessary stuffs.